recipes/clyngor/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "clyngor" %}
+{% set version = "0.3.30" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bca099abe4f97ad5f4b9270721b6047737414b14e5c50c1d4f97a2339e1b8cda
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - pypeg2
+    - arpeggio
+  run:
+    - python
+    - arpeggio
```

Comment:
There are version restrictions on `arpeggio` and `pypeg2`:

https://github.com/Aluriak/clyngor/blob/master/setup.cfg#L26-L27